ibis/backends/dask/__init__.py

```diff
@@ -1,66 +1,17 @@
-from typing import Dict
-
-from dask.dataframe import DataFrame
-
 import ibis.config
-from ibis.backends.base import BaseBackend
+from ibis.backends.pandas import BasePandasBackend
 
 from . import udf  # noqa: F401,F403 - register dispatchers
 from .client import DaskClient, DaskDatabase, DaskTable
 from .execution import execute  # noqa F401
 
+# Make sure that the pandas backend is loaded, dispatching has been
+# executed, and options have been loaded
+ibis.pandas
+
 
-class Backend(BaseBackend):
+class Backend(BasePandasBackend):
     name = 'dask'
-    kind = 'pandas'
     database_class = DaskDatabase
     table_class = DaskTable
-
-    def connect(self, dictionary: Dict[str, DataFrame]) -> DaskClient:
-        """Construct a dask client from a dictionary of DataFrames.
-
-        Parameters
-        ----------
-        dictionary : dict
-
-        Returns
-        -------
-        DaskClient
-        """
-        return DaskClient(backend=self, dictionary=dictionary)
-
-    def from_dataframe(
```

Comment:
where does this go?

Reply:
This is the same exact code as the same method in pandas. I created a base pandas backend class in the pandas backend, and both the pandas and the dask backend subclass it. So we can get rid of a bit of the code that was copied from pandas into dask and reuse it.